.pre-commit-config.yaml

```diff
@@ -9,11 +9,15 @@ repos:
   - id: yapf
     args: [--in-place, --verbose]
     additional_dependencies: [toml] # TODO: Remove when yapf is upgraded
+    exclude: "^(csrc|vllm/assets|vllm/inputs|vllm/multimodal|vllm/usage)/.*"
```

Comment:
Could this exclude be moved to the `pyproject.toml`? Generally we prefer to avoid configuration that only works if the formatting tools are run via pre-commit (i.e. we would like format on save to work correctly)